src/convertMswMatchToPact.msw.spec.ts

```diff
@@ -32,7 +33,7 @@ const sampleMatch: MswMatch[] = [
   {
     request: {
       id: 'de5eefb0-c451-4ae2-9695-e02626f00ca7',
-      url: new URL('http://localhost:8081/products'),
+      url: new URL('http://localhost:8081/products?sort=asc'),
```

Comment:
could we potentially create a new test here, and parameterise these so we can expect the query, or not expect the query in a generated pact file conversion please?